### ISSUE TITLE
pagination rowsPerPageOptions not working

### ISSUE THREAD

**Issue by johncalvinroberts** [marmelab/react-admin#2673]
**What you were expecting:**
Hi! Thanks for the awesome framework!

I am following the documentation, trying to allow a resource to have a selection of perPage options. Followed this snippet verbatim from the documentation:
```javascript
import { Pagination } from 'react-admin';

const PostPagination = props => <Pagination rowsPerPageOptions={[10, 25, 50, 100]} {...props} />

export const PostList = (props) => (
    <List {...props} pagination={<PostPagination />}>
        ...
    </List>
);
```

**What happened instead:**
No `perPage` options are rendered, the `pagination` component stays the same, error is logged to console:
```Warning: React does not recognize the `rowsPerPageOptions` prop on a DOM element.```

Unable to select any perpage options, pagination component does not change or accept this prop.

**Steps to reproduce:**
Follow this: https://github.com/marmelab/react-admin/blob/master/docs/List.md#pagination

**Environment**

* React-admin version: 2.1.1
* Last version that did not exhibit the issue (if applicable):
* React version: 16.3.2
* Browser: Chrome 71
* Stack trace (in case of a JS error):

Is there a reason this doesn't work in this environment? Is this a bug or are the docs out of date? Thanks in advance!

**Comment by djhi** [comment]
That's because this feature was released in [v2.3.1](https://github.com/marmelab/react-admin/releases/tag/v2.3.1).

**Comment by johncalvinroberts** [comment]
Thank you @djhi
